controllers/components/feastoperator/feastoperator_support.go

```diff
@@ -0,0 +1,38 @@
+package feastoperator
+
+import (
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+
+	componentApi "github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1alpha1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/controllers/status"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/types"
+	odhdeploy "github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+)
+
+const (
+	ComponentName = componentApi.FeastOperatorComponentName
+
+	ReadyConditionType = conditionsv1.ConditionType(componentApi.FeastOperatorKind + status.ReadySuffix)
+
+	// LegacyComponentName is the name of the component that is assigned to deployments
+	// via Kustomize. Since a deployment selector is immutable, we can't upgrade existing
+	// deployment to the new component name, so keep it around till we figure out a solution.
+	LegacyComponentName = "feastoperator"
+
+	ManifestsSourcePath = "overlays/odh"
+)
+
+var (
+	imageParamMap = map[string]string{
+		"RELATED_IMAGE_FEAST_OPERATOR": "RELATED_IMAGE_ODH_FEAST_OPERATOR_IMAGE",
```

Comment:
You may review the naming. Usually "RELATED" is for environment variables name (values here) and image have some lowercase pure names in your params.env and kustomization.

Reply:
> WDYT if still need to change this into small naming, if yes then are these better names ?
No, fine for me. I just asked since in many components pattern is different. I did not check the details. Thanks for the explanation.